lib/Transform/ATenVisualGraph.cpp

```diff
@@ -806,8 +806,10 @@ struct ATenVisualGraphPass : public ATenVisualGraphBase<ATenVisualGraphPass> {
       opInput = getInput(op2);
     } else if (auto op2 = mlir::dyn_cast<xten::Conv2dReLUPadMaxPoolOp>(op)) {
       opInput = getInput(op2);
+    } else if (auto op2 = mlir::dyn_cast<Torch::AtenFlattenUsingIntsOp>(op)) {
```

Comment:
If we ever refactor this code, this huge if-else  that does basically nothing is the very first thing to get deleted.